custom_dc/build_image.md

```diff
@@ -41,84 +41,68 @@ If you want to pick up the latest prebuilt version, do the following:
-- You have a Github account and project.
-- You have created a fork off the base Data Commons `website` repo (https://github.com/datacommonsorg/website){: target="_blank"} and a remote that points to it, and that you will push to that fork. 
-
-
-### Sync a local workspace to the stable release
+Rather than building from the master branch, which includes the very latest changes in Github, that may not have been tested, we recommend that you use the tested "stable" branch equivalent of the stable Docker image. This branch is `customdc_stable`, and is available at [https://github.com/datacommonsorg/website/tree/customdc_stable](https://github.com/datacommonsorg/website/tree/customdc_stable){: target="_blank"}.
 
-If you are using a version control system other than Github, you can download a ZIP or TAR file from [https://github.com/datacommonsorg/website/releases/tag/customdc_stable](https://github.com/datacommonsorg/website/releases/tag/customdc_stable){: target="_blank"}. 
+> **Note:** If you are working on a large-scale customization, we recommend that you use a version control system to manage your code. We provide some procedures for Github.
 
-In Github, use the following procedure.
+#### Clone the stable branch only
 
-1. If you want to reuse the root directory you previously created and cloned, skip to step 3. 
-If you want to create a new source directory and start from scratch, clone the repo up to the stable release tag:
+Use this procedure if you are not using Github, or if you are using Github and want to create a new source directory and start from scratch.
 
+1. Run the following command:
    <pre>
    git clone https://github.com/datacommonsorg/website --branch customdc_stable --single-branch  [<var>DIRECTORY</var>]
    </pre>
-1. Change to the root directory:
-
+   This creates a new local branch called `customdc_stable` set to track the Data Commons repo branch. 
+1. To verify, run:
    <pre>
    cd website | cd <var>DIRECTORY</var>
+   git branch -vv
    </pre>
+   You should see output like the following:
+
+   ```
+   * customdc_stable 83732891 [origin/customdc_stable] 2024-11-06 Custom DC stable release (#4710)
+   ```
+   Rather than developing on this default branch, we recommend that you create another branch.
 
-1. Create a new branch synced to the stable release:
+#### Sync code to the stable branch
 
+The following procedure uses Github. If you are using another version control system, use the appropriate methods for updating submodules and syncing.
+
+1. Switch to the directory where you have cloned the Data Commons doe:
    <pre>
-   git checkout -b <var>BRANCH_NAME</var> customdc_stable
+   cd website | cd <var>DIRECTORY</var>
    </pre>
 
-1. To verify that your local repo is at the same version of the code, run the following command:
-
+1. Update files:
    ```
-   git log --oneline --graph
+   git pull origin customdc_stable
```

Comment:
Can you confirm that since they have only cloned the stable branch there's no need to do `git checkout origin/customdc_stable`?

Reply:
So this is the part we took out -- which is creating a new branch and working off that. If they use your command, they get into detached HEAD mode, which they don't want. What they want is git checkout -b <newbranchname> origin/customdc_stable, but we agreed to remove that step? (I'm happy to put it back in :-))

Reply:
This works as-is! I just tested and since they only clone the one branch, there's no need to also check out the branch (it's checked out by default).